app/flatpak-builtins-enter.c

```diff
@@ -79,7 +80,7 @@ flatpak_builtin_enter (int           argc,
   uid_t uid;
   gid_t gid;
 
-  context = g_option_context_new (_("SANDBOXEDPID [COMMAND [args...]] - Run a command inside a running sandbox"));
+  context = g_option_context_new (_("PID-OR-APP [COMMAND [args...]] - Run a command inside a running sandbox"));
```

Comment:
Would be nice to accept an intance id here too.

Reply:
I'll look into that